src/schema-to-typebox.ts

```diff
@@ -317,6 +317,12 @@ export const parseTypeName = (
     return schemaOptions === undefined
       ? "Type.Null()"
       : `Type.Null(${schemaOptions})`;
+  } else if (type === "object") {
+    return parseObject(schema as ObjectSchema)
+  // We don't want to trust on build time checking here, json can contain anything
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```

Comment:
Interesting. This error should have been there in the previous version to but somehow with the utitility type it did not recognize it :D 

Will create separate PR to get this merged, the reason is noted down [here](https://github.com/xddq/schema2typebox/pull/47) and will be in the new PR.